README.md

```diff
@@ -112,6 +112,25 @@ docker-compose -f simplewallet-build-tp-<your_prog_language>.yaml up --build
 where,
  <your_prog_language> should be replaced with either `cxx`, `java`, or `py`
 
+# Building and running on OS(without dockers)
+To run sawtooth-simplewallet without dockers, we'll have to use a Ubuntu 16.04 OS installation and compile simplewallet from sources. Below is a sample procedure for python TP/client:
+
+1. Install sawtooth on Ubuntu 16.04 LTS x64 machine. Refer sawtooth app developer's guide [here](https://sawtooth.hyperledger.org/docs/core/releases/latest/app_developers_guide/ubuntu.html)
+2. git clone https://github.com/askmish/sawtooth-simplewallet.git
+3. `cd sawtooth-simplewallet`
+4. Setup the simplewallet-tp.
+ - Follow the [`pyprocessor/Dockerfile`](https://github.com/askmish/sawtooth-simplewallet/blob/master/pyprocessor/Dockerfile)
+ - Install all the dependencies in the [first `RUN` line](https://github.com/askmish/sawtooth-simplewallet/blob/master/pyprocessor/Dockerfile#L18) in the Dockerfile
```

Comment:
- Start the validator, REST API, and Settings TP in separate consoles:
`sudo -u sawtooth sawtooth-validator -vv`
`sudo -u sawtooth sawtooth-rest-api -vvv`
`sudo -u sawtooth settings-tp -vv`